history/provinces/00_k_lordaeron.txt

```diff
@@ -717,9 +735,16 @@
 	culture = lordaeronian
 	religion = holy_light
 	holding = castle_holding
+	100.1.1 = {
+		special_building_slot = scarlet_monastery_01
```

Comment:
Why in Hearthglen ?

Reply:
When I was looking for the location it was north of the Venomweb Vale, which in the game is Hearthglen

Reply:
<details>![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/46578034/3db44b8c-77fd-4300-a18a-0dca79f30001)</details>

Clearly not at the same place

Reply:
yeah ok you're right
I'll put it in the Whispering Gardens